invokeai/frontend/web/src/features/stylePresets/components/StylePresetImageField.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Button, Flex, Icon, IconButton, Image, Tooltip } from '@invoke-ai/ui-library';
+import { useCallback } from 'react';
+import { useDropzone } from 'react-dropzone';
+import type { UseControllerProps } from 'react-hook-form';
+import { useController } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { PiArrowCounterClockwiseBold, PiUploadSimpleBold } from 'react-icons/pi';
+
+import type { StylePresetFormData } from './StylePresetForm';
+
+export const StylePresetImageField = (props: UseControllerProps<StylePresetFormData>) => {
```

Comment:
Pass the field name as the second type arg in these generics to get proper types:
```suggestion
export const StylePresetImageField = (props: UseControllerProps<StylePresetFormData, 'image'>) => {
```

Then remove the type casts below.